package-lock.json

```diff
@@ -1281,6 +1281,17 @@
         }
       }
     },
+    "@rpl/badge-up": {
```

Comment:
this is a lot of new dependencies 🤔 (634 new lines) is it just from `flow-coverage-report`?

Reply:
Agreed. Removed the dependency because of this.